### PR TITLE
[sw] remove entropy_src work arounds for aes

### PR DESCRIPTION
**Pull request by tjaychen**
Signed-off-by: Timothy Chen <timothytim@google.com>

### CODE REVIEW

File: sw/device/tests/aes_test.c [hunk 1]
```diff
@@ -90,14 +90,6 @@ bool test_main(void) {
...
   }
 
-  // Disable and re-enable EDN0 to get some more entropy out of it. This is
-  // dirty and needs to be reworked. We need to setup CSRNG/EDN to continously
-  // provide entropy.
-  mmio_region_write32(mmio_region_from_addr(TOP_EARLGREY_EDN0_BASE_ADDR),
-                      EDN_CTRL_REG_OFFSET, 0x0);
-  mmio_region_write32(mmio_region_from_addr(TOP_EARLGREY_EDN0_BASE_ADDR),
-                      EDN_CTRL_REG_OFFSET, 0x1);
-
```
Comment (mwbranstad) [inline sw/device/tests/aes_test.c:100]: With pr #5474 this change is not needed anymore.

**Review by mwbranstad — APPROVED** [review]
LGTM

**Review by vogelpi — APPROVED** [review]
Thanks @tjaychen  , this LGTM!

However, I can imagine that once more devices will start to use entropy other people will run into the same problem again at a later point. And they will not know what to do. So, I am wondering if we should just leave this as is until the entropy complex is configured to provide continuous entropy?

**Review by mcy — APPROVED** [review]

**Comment by tjaychen** [comment]
> Thanks @tjaychen , this LGTM!
> 
> However, I can imagine that once more devices will start to use entropy other people will run into the same problem again at a later point. And they will not know what to do. So, I am wondering if we should just leave this as is until the entropy complex is configured to provide continuous entropy?

so i think for individual one-off tests, the amount of entropy that can be supplied via entropy_src will probably always be enough (need to double check exactly how much).

I think eventually when entropy_enable goes into boot rom we will need to have a much closer look.

**Pull request merged**